test/Sentry/SentryLaravelServiceProviderTest.php

```diff
@@ -20,7 +17,7 @@ protected function getPackageProviders($app)
     protected function getPackageAliases($app)
     {
         return [
-            'Sentry' => 'Sentry\SentryLaravel\SentryFacade'
+            'Sentry' => 'Sentry\SentryLaravel\SentryFacade',
```

Comment:
Maybe you can use `::class`?

Reply:
No since we have to support PHP 5.4 :( (for Laravel 5.0)